src/main/java/org/mastodon/views/trackscheme/display/TrackSchemeFrame.java

```diff
@@ -64,7 +64,7 @@ public TrackSchemeFrame(
 			final TrackSchemeGraph< ?, ? > graph,
 			final HighlightModel< TrackSchemeVertex, TrackSchemeEdge > highlight,
 			final FocusModel< TrackSchemeVertex, TrackSchemeEdge > focus,
-			final TimepointModel timepoint,
+			final TimepointModel navigationTimepointModel, final TimepointModel fadingTimepointModel,
```

Comment:
Please add a new line after the comma, to keep the rule of one parameter per line.

Reply:
Awesome. I find this so much better now. That we don't need two timepoint models here.